proposals/831-helm-support/README.MD

```diff
@@ -0,0 +1,454 @@
+
+### About Helm
+
+- Helm is a graduated project in the CNCF and is maintained by the Helm community.
+- Helm is supported by and built with a community of over 400 developers.
+- Helm helps you manage Kubernetes applications, rollbacks, updates, dependencies, and releases, and the most complex Kubernetes applications. [More about Helm](https://helm.sh/)
+
+## Value to the Community
+
+- Streamline the deployment, upgrade, and rollback of the kubeflow installation process
+- Provide another way to install Kubeflow to give the community more options, promoting flexibility and choice
+
+## Goals
+
+
+✅ A fully functional Kubeflow Helm chart for the targeted release. This will install Kubeflow as a platform and as an individual component.   
+✅ Published Helm chart documentation with straightforward and uncomplicated configuration options.   
+✅ A step-by-step tutorial simplifying Kubeflow deployment for users.   
+✅ Contribution to the Kubeflow community effort for Helm-based installation as part of the official Kubeflow repository.   
+✅ Where possible, rely on upstream Helm charts (i.e., KServe/Istio).   
+✅ Consolidate community Helm efforts and prevent duplicate efforts.  
+
+
+## Non-Goals
+* Deep integration with hyperscalers such as AWS managed databases. Nevertheless for example basic Dex/oauth2-proxy configuration for authentication integration with popular Kubernetes platforms such as EKS is a goal, because it is needed for M2M authentication within Kubeflow.
+* Infrastructure provisioning. Users can opt into OpenTofu or Crossplane to template the Helm chart with infrastructure. Still, we are focused on the Helm chart where the values would be configured (if a component requires knowledge of external systems).
+* Support separate abstracted operators for components. Helm will handle upgrades.
+* Provide guaranteed community support. 
+* Define production for any particular set of users.
+
+
+## Proposal
+This proposal introduces official Helm chart support for deploying Kubeflow. The goal is to provide a modular, community-maintained method for installing and managing Kubeflow, making it more accessible for users who prefer Helm over Kustomize-based manifests.
+
+
+## Desired Outcome
+
+
+The Helm chart will allow users to: 
+
+
+✅ Deploy Kubeflow with a single Helm command, reducing installation complexity.   
+✅ Select specific components to install (e.g., Training Operator, Katib, Pipelines) without requiring the entire Kubeflow stack.   
+✅ Configure installations via Helm values, enabling customization for different environments (e.g., resource allocation, authentication settings, storage options).   
+✅ Upgrade and rollback Kubeflow deployments safely using Helm’s built-in version control.   
+✅ Integrate with GitOps workflows (e.g., ArgoCD, FluxCD) for automated deployments.   
```

Comment:
I don't think we should have integration with GitOps workflows, since it is out-of-scope.

Reply:
Yes, let's remove "Integrate with GitOps workflows (e.g., ArgoCD, FluxCD) for automated deployments.   "